lib/facade_converter.ts

```diff
@@ -246,8 +246,7 @@ export class FacadeConverter extends base.TranspilerBase {
     'lib.es6': this.stdlibTypeReplacements,
     'angular2/typings/es6-promise/es6-promise': {'Promise': 'Future'},
     'angular2/typings/es6-shim/es6-shim': {'Promise': 'Future'},
-    'angular2/src/facade/async':
-        {'Observable': 'Stream', 'ObservableController': 'StreamController'},
+    'rx/Observable': {'Observable': 'Stream'},
```

Comment:
This should be `rxjs/Observable`.  

But it is actually worse than that. It looks like because we do not use the node resolution strategy, typescript makes the path to rxjs/Observable relative. So the working configuration looks like this:

```
'../../../../../../rxjs/Observable': { 'Observable': 'Stream' },
```